pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/services/RecorderPlaybackService.scala

```diff
@@ -42,21 +43,19 @@ class RecorderPlaybackService(
     * If the work is missing from VHS, it throws [[NoSuchElementException]].
     */
   private def getWorkForIdentifier(
-    workIdentifier: WorkIdentifier): Future[Option[TransformedBaseWork]] = {
-    workIdentifier.version match {
-      case 0 => Future.successful(None)
-      case _ =>
-        versionedHybridStore.getRecord(id = workIdentifier.identifier).map {
-          case None =>
-            throw new NoSuchElementException(
-              s"Work ${workIdentifier.identifier} is not in VHS!")
-          case Some(work) if work.version == workIdentifier.version =>
-            Some(work)
-          case Some(work) =>
-            debug(
-              s"VHS version = ${work.version}, identifier version = ${workIdentifier.version}, so discarding work")
-            None
+    workIdentifier: WorkIdentifier): Option[TransformedBaseWork] =
+    vhs.getLatest(workIdentifier.identifier) match {
+      case Right(Identified(Version(_, version), HybridStoreEntry(work, _))) =>
```

Comment:
Do you use the captured `version`, or can you `_` the entire Version()?

Reply:
:+1: